backend/python/app/graphql/queries/story_query.py

```diff
@@ -5,14 +5,17 @@
 from ..service import services
 
 
+@require_authorization_by_role_gql({"User", "Admin"})
 def resolve_stories(root, info, story_title, start_date, end_date, **kwargs):
```

Comment:
Not sure if this can be admin only.

Reply:
hmmm, `test_stories` in `python\app\tests\graphql\queries\test_story_query.py` actually fails since we dont have a way of authenticating users via our gql tests. Can you comment out the body of the test fn with a block comment (ie 
```
""" block comment """
```
and also leave a comment that the test doesnt work anymore because the query is authenticated, and we dont have a way of testing authenticated queries lol

Reply:
(we actually dont use this query anywhere in teh frontend oops lol)